flake.nix

```diff
@@ -467,7 +467,7 @@
             cargoArtifacts = deps;
 
             src = filterModules dirs ./.;
-            cargoExtraArgs = "--package ${name} --target ${target.name}";
+            cargoExtraArgs = "--target ${target.name} --package ${name}";
```

Comment:
Just style change, in case you were wondering. To match the one slightly above building the deps.